src/components/modals/item-modal/PlaceBidModal.tsx

```diff
@@ -51,8 +57,20 @@ const PlaceBidModal = ({ show, handleModal, item }: Props) => {
             </p>
             <div className="placebid-form-box">
               <div className="bit-continue-button">
-                <Button size="medium" fullwidth onClick={onSubmit}>
-                  Buy NFT for {item?.price} ETH
+                <Button
+                  size="medium"
+                  fullwidth
+                  onClick={onSubmit}
+                  className={isLoading ? 'disabled' : ''}
+                >
+                  {isLoading ? (
```

Comment:
This is cool. However, I'd suggest to create a reusable spinner button component, since we will use it all over the place. (not required now, it is worth to create a new task in the board though)

Reply:
Sure, I'll create it